ddl/ddl_workerpool_test.go

```diff
@@ -27,7 +27,7 @@ func TestBackfillWorkerPool(t *testing.T) {
 			return wk, nil
 		}
 	}
-	pool := newDDLWorkerPool(pools.NewResourcePool(f(), 1, 2, 0), reorg)
```

Comment:
pools.NewResourcePool have an plan to remove it. it is just an Object Pool. It will be replaced with ```sync.Pool```.